core/src/main/java/bio/terra/pearl/core/service/publishing/PortalPublishingService.java

```diff
@@ -81,7 +81,7 @@ protected PortalEnvironment applyUpdate(PortalEnvironment destEnv, PortalEnviron
 
         applyChangesToPreRegSurvey(destEnv, envChanges.preRegSurveyChanges());
         applyChangesToSiteContent(destEnv, envChanges.siteContentChange());
-        applyChangesToNotificationConfigs(destEnv, envChanges.notificationConfigChanges());
+        applyChangesToNotificationConfigs(destEnv, envChanges.triggerChanges());
```

Comment:
`applyChangesToNotificationConfigs` should be updated to `applyChangesToTriggers`